spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -628,7 +633,12 @@ def setup_trailers
 
       # generates the nested data structure to represent trailers
       def setup_trailers_for(row)
-        display_families = row.fetch("displayFamilies", {}).fetch("value", nil)
+        return if row.nil?
+
+        display_families_hash = row.fetch("displayFamilies", {})
+        return {} unless display_families_hash
```

Comment:
Hm, I don't think we want this method to possibly return `{}`.

With your change  below, I think you can adjust the line above to be:

```ruby
display_families_hash = row.fetch("displayFamilies", {}) || {}
```

and remove the line starting with `return`